steps/sheepdogAPI.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+let username = process.env.INDEX_USERNAME;
```

Comment:
can you make the code getting username, password, and creating token in the headers as a util function? Don't repeat the code in different places